bin/chat_server_log.log

```diff
@@ -0,0 +1,4192 @@
+Apr 04, 2017 5:37:42 PM codeu.chat.ServerMain main
```

Comment:
Ditto for .class and .lck files below. Source control doesn't excel at handling binary files, and it just makes the pull requests larger.